<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UncertaintyPlannerDlgUI</class>
 <widget class="QWidget" name="UncertaintyPlannerDlgUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sampleButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>sample</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>object</string>
   </property>
  </widget>
  <widget class="QPushButton" name="visualizeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>visualize</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadUncertaintyButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>uncertainty</string>
   </property>
  </widget>
  <widget class="QSlider" name="percentageSlider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>240</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="percentageLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>240</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>n/a</string>
   </property>
  </widget>
  <widget class="QSlider" name="intervalSlider">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>160</width>
     <height>18</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="intervalLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>n/a</string>
   </property>
  </widget>
  <widget class="QPushButton" name="marchingCubeButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>261</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Mesh it</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
